src/components/Icon/Icon.css

```diff
@@ -12,6 +12,8 @@
 
 .y-icon__isBlock {
   display: block;
+  position: static;
+  margin-top: 0.2rem;
```

Comment:
I was torn about adding the margin here.  Unless I'm missing something the alternative was to add wrapping blocks with push=2, to many uses of Icon.

Reply:
this actually won't work because we're adding 2px to the overall height of the Icon now. there are some tricky bits regarding #327, i'd suggest converting CSS first and then tackling #327 later (i'm not sure what the best solution is, we can put our heads together on it later)